File: app/src/main/java/com/eagskunst/emmanuel/gamingnews/views/ArticlesFromNotificationActivity.java
package com.eagskunst.emmanuel.gamingnews.views;

import android.os.Bundle;
import android.support.v7.widget.Toolbar;

import com.eagskunst.emmanuel.gamingnews.fragments.NewsListFragment;
import com.eagskunst.emmanuel.gamingnews.R;
import com.eagskunst.emmanuel.gamingnews.utility.BaseActivity;

public class ArticlesFromNotificationActivity extends BaseActivity {

    private static final String TAG = "ArticlesFromNotification";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_articles_from_notification);
        Toolbar toolbar = findViewById(R.id.toolbar);
        showToolbar(toolbar,R.string.notification,true,null);
        callLog(TAG, "Title: " + getSupportActionBar().getTitle().toString());
        String[] fakeUrls = {TAG};
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.notificationContainer,NewsListFragment.newInstance(fakeUrls))
                .commit();
    }
}

File: app/src/main/java/com/eagskunst/emmanuel/gamingnews/views/ArticlesFromNotificationActivity.kt
package com.eagskunst.emmanuel.gamingnews.views

import android.os.Bundle
import androidx.appcompat.widget.Toolbar
import com.eagskunst.emmanuel.gamingnews.R
import com.eagskunst.emmanuel.gamingnews.fragments.news_list.NewsListFragment
import com.eagskunst.emmanuel.gamingnews.utility.BaseActivity

class ArticlesFromNotificationActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_articles_from_notification)
        val toolbar: Toolbar = findViewById(R.id.toolbar)
        showToolbar(toolbar, R.string.notification, true, null)
        callLog(TAG, "Title: " + supportActionBar!!.title.toString())
        val fakeUrls = arrayOf(TAG)
        supportFragmentManager.beginTransaction()
            .replace(R.id.notificationContainer, NewsListFragment.newInstance(fakeUrls))
            .commit()
    }

    companion object {
        private const val TAG = "ArticlesFromNotification"
    }
}
